bot/exts/evergreen/quack.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional
+
+import discord
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours
+
+API_URL = 'https://quackstack.pythondiscord.com'
+
+
+class Quack(commands.Cog):
+    """Cog for the .quack command."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @commands.command()
+    async def quack(self, ctx: commands.Context, ducktype: Optional[str] = "duck", seed: Optional[str] = None) -> None:
+        """
+        Uses the Quackstack API to generate a random duck.
+
+        If a seed is provided, a duck is generated based on the given seed.
+        If a duck type is provided, either a duck or manduck will be generated.
+        """
+        ducktype = ducktype.lower()
+        if ducktype not in ["duck", "manduck"]:
+            raise commands.BadArgument(
+                "The given duck type is invalid. The duck type can either be `duck` or `manduck`."
```

Comment:
Just noticed, as it adds on a bit to the front here it's a little repetitive, could probably remove the "The given duck type is invalid" bit 

![Discord_ueMwIE1iRa](https://user-images.githubusercontent.com/22353562/132133785-25a30f58-123a-49f0-97a5-c8baa194a383.png)